### ISSUE TITLE
keyup and keydown not working on masOs

### ISSUE THREAD

**Issue by jhabril** [shoes/shoes3#462]

**Comment by ccoupe** [comment]
Yes. Shoes does not handle keyup/keydown on OSX. Cocoa provides keydown and keyup events, but not the 'keypressed' distinction of Shoes. So, osx.keydown had to map to Shoes.keypress.  It's not likely to be fixed. Sorry.

**Comment by jhabril** [comment]
Can keyup event be handle by other method or code?

**Comment by ccoupe** [comment]
It takes an Objective C programmer who know Cocoa and Shoes internals. There is no work around.
